src/header/navigation.rs

```diff
@@ -29,10 +29,10 @@ pub fn Header() -> impl IntoView {
 
     view! {
         <header class="z-10 bg-main-background flex justify-center items-center md:grid md:grid-cols-[10%_20%_60%_10%] md:col-start-2 md:col-end-3 fixed top-0 left-0 w-screen h-16">
-            <div class="md:col-start-2 md:col-end-3 flex items-center justify-start">
+            <a href="/summary" class="md:col-start-2 md:col-end-3 flex items-center justify-start">
```

Comment:
This should link to `/` rather than `/sumamry`

Reply:
Yes